proxyclient/m1n1/asm.py

```diff
@@ -72,10 +37,10 @@ def compile(self, source):
         self.bfile = self._tmp + "b.b"
         self.nfile = self._tmp + "b.n"
 
-        self._call(CC, f"{self.CFLAGS} -c -o {self.ofile} {self.sfile}")
-        self._call(LD, f"{self.LDFLAGS} --Ttext={self.addr:#x} -o {self.elffile} {self.ofile}")
-        self._call(OBJCOPY, f"-j.text -O binary {self.elffile} {self.bfile}")
-        self._call(NM, f"{self.elffile} > {self.nfile}")
+        self._call(self.toolchain.CC, f" -c -o {self.ofile} {self.sfile}")
+        self._call(self.toolchain.LD, f" --Ttext={self.addr:#x} -o {self.elffile} {self.ofile}")
+        self._call(self.toolchain.OBJCOPY, f" -j.text -O binary {self.elffile} {self.bfile}")
```

Comment:
```suggestion
        self._call(self.toolchain.CC, f"-c -o {self.ofile} {self.sfile}")
        self._call(self.toolchain.LD, f"--Ttext={self.addr:#x} -o {self.elffile} {self.ofile}")
        self._call(self.toolchain.OBJCOPY, f"-j.text -O binary {self.elffile} {self.bfile}")
```

no need for the odd leading space.